src/main/java/org/jabref/logic/bibtex/comparator/BibDatabaseDiff.java

```diff
@@ -83,28 +69,26 @@ private static List<BibEntryDiff> compareEntries(List<BibEntry> originalEntries,
         }
 
         // Now we've found all exact matches, look through the remaining entries, looking for close matches.
-        for (Iterator<BibEntry> iteratorNotMatched = notMatched.iterator(); iteratorNotMatched.hasNext(); ) {
-            BibEntry originalEntry = iteratorNotMatched.next();
-
+        DuplicateCheck duplicateCheck = new DuplicateCheck(Globals.entryTypesManager);
```

Comment:
This wont work. The EntryTypesManager needs to be injected in the method arguments. And Globals is a deprecated GUI only class.